handlers/addPokemonRoute.js

```diff
@@ -0,0 +1,37 @@
+const PokemonData = require('../models/pokemons');
+
+const addPokemon = {
+    method: 'POST',
+    handler: async function (request, h) {
+        const newPokemon = request.payload; 
+
+        if (!newPokemon.id) {
```

Comment:
This validation has a wrong format. It should be (pokemon.id == null)

Reply:
We are using joi to solve that.